apollo-backend/src/main/java/io/logz/apollo/services/SlaveService.java

```diff
@@ -63,6 +63,8 @@ public SlaveService(ApolloConfiguration apolloConfiguration, SlaveDao slaveDao,
 
     @PostConstruct
     public void start() {
+        logger.info("Starting slave.. slaveId - {}, environmentIds - {}", slaveId, environmentIds);
```

Comment:
You need to join the environmentIds, or use apolloConfiguration.getSlave().getSlaveCsvEnvironments();